extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -208,7 +208,8 @@ export class NetCoreTool extends ShellExecutionHelper {
 			} else if (this.netCoreInstallState === netCoreInstallState.netCoreVersionTooHigh && vscode.workspace.getConfiguration(DBProjectConfigurationKey)[NetCoreDowngradeDoNotShowAgainKey] === true) {
 				// Assume user has used global.json to override SDK version and proceed with build as is
 			} else {
-				throw new DotNetError(NetCoreSupportedVersionInstallationConfirmation(this.netCoreSdkInstalledVersion!));
+				// don't continue building if the user hasn't selected the do not show again on the warnings shown from findOrInstallNetCore()
+				return '';
```

Comment:
This is a generic "run dotnet command" function so I don't think having build-specific comments here is the best. 

And do you really want to just return here and not throw an error? Returning would indicate to me that it completed successfully which isn't the case.

Reply:
+1 to both

Reply:
The error message was already a project build specific message, but I added it back and changed the build function that calls it to not show the error again if it's a dotnet error since it already gets shown by the netCoreHelper